enclave/cert.c

```diff
@@ -731,31 +745,23 @@ oe_result_t oe_cert_verify(
     oe_cert_t* cert,
     oe_cert_chain_t* chain,
     const oe_crl_t* const* crls,
-    size_t num_crls,
-    oe_verify_cert_error_t* error)
+    size_t num_crls)
 {
     oe_result_t result = OE_UNEXPECTED;
     Cert* cert_impl = (Cert*)cert;
     CertChain* chain_impl = (CertChain*)chain;
-    uint32_t flags = 0;
     mbedtls_x509_crl* crl_list = NULL;
 
-    /* Initialize error */
-    if (error)
-        *error->buf = '\0';
-
     /* Reject invalid certificate */
     if (!_cert_is_valid(cert_impl))
     {
-        _set_err(error, "invalid cert parameter");
-        OE_RAISE(OE_INVALID_PARAMETER);
+        OE_RAISE_MSG(OE_INVALID_PARAMETER, "Invalid cert parameter", NULL);
```

Comment:
The final NULL parameter is not needed.

Reply:
Technically true because it currently only compiles on linux, but MSVC flags all such instances on the host side and so I've gotten into the habit of requiring explicit NULL for the macro.

Reply:
This seems like a flaw with **OE_RAISE_MSG()**. Maybe we can fix as a separate issue.